backend/src/models/__test__/TestDatabase.ts

```diff
@@ -0,0 +1,60 @@
+import 'reflect-metadata'
+
+process.env.NODE_ENV = 'test'
+
+import { config } from 'dotenv'
+config()
+
+import { createConnection, ConnectionOptions, Connection } from 'typeorm'
+import { createServer, Server as HttpServer } from 'http'
+
+import express from 'express'
+import supertest from 'supertest'
+
+import { app } from '../../index'
+
+import { User, Expense } from '..'
+
+export class TestDatabase {
+  private _app: express.Application
+  private _connection: Connection
+  private _server: HttpServer
+
+  private connectionOptions: ConnectionOptions = {
+    type: 'postgres',
+    port: parseInt(process.env.PORT_DB),
+    username: process.env.USERNAME_DB,
+    password: process.env.PASSWORD_DB,
+    database: 'expense_manager_test',
+    entities: [User, Expense],
+    synchronize: true,
+    logging: false,
+  }
```

Comment:
@Joboa make sure to create an `expense_manager` (production) and `expense_manager_test` locally in Postgres.

```
$ psql
#= CREATE DATABASE expense_manager;
#= CREATE DATABASE expense_manager_test;
```

Reply:
Okay, I will do that.